examples/events/main.rs

```diff
@@ -0,0 +1,97 @@
+extern crate amethyst;
+
+use amethyst::core::{
+    bundle::SystemBundle,
+    frame_limiter::FrameRateLimitStrategy,
+    shrev::{EventChannel, ReaderId},
+};
+use amethyst::{
+    ecs::{DispatcherBuilder, Read, Resources, System, SystemData, World, Write},
+    prelude::*,
+};
+
+use amethyst::Error;
+use core::result::Result;
+
+#[derive(Debug)]
+struct MyBundle;
+
+impl<'a, 'b> SystemBundle<'a, 'b> for MyBundle {
+    fn build(self, builder: &mut DispatcherBuilder<'a, 'b>) -> Result<(), Error> {
+        builder.add(SpammingSystem, "spamming_system", &[]);
+        builder.add(
+            ReceivingSystem {
+                reader: std::prelude::v1::Option::None,
```

Comment:
Why not just Option::None?

Reply:
Good catch! I was converting it from my scrapbook where I simply copy/pasted suggestion from the compiler. Thanks, updating :).